spec/spec_helper.rb

```diff
@@ -29,6 +28,8 @@
 require_relative "support/hint_examples.rb"
 require_relative "support/transformation_examples.rb"
 
+require 'alchemy/test_support/essence_shared_examples'
```

Comment:
I am fine with this, if it's placed after https://github.com/watg/alchemy_cms/blob/move_shared_examples_after_rspec/spec/spec_helper.rb#L26

@mtomov could you please try to put it on L27 and see if it's still working for you? Thanks :)